graaljs/graaljs-gradle-webpack-guide/README.md

```diff
@@ -0,0 +1,187 @@
+# Using Node Packages in a Java Application with Gradle
+
+## Introduction
+This guide demonstrates how to integrate JavaScript libraries into a Java application using **GraalJS** and the **GraalVM Polyglot API**. Specifically, we will use the `qrcode` NPM package to generate QR codes within a Java application, leveraging GraalVM's ability to execute JavaScript code.
+
+## Prerequisites
+Before starting, ensure you have the following installed:
+- **JDK 21** or later
+- **Gradle 7.0** or later
+- A text editor or IDE of your choice
+
+## Installation
+
+### Step 1: Clone the Repository
+Clone the project repository to your local machine:
+
+```sh
+git clone https://github.com/graalvm/graal-languages-demos.git
+cd graaljs/graaljs-gradle-webpack-guide
+```
+
+### Step 2: Compile and Run the Java Application
+To build and run the Java application, use the following Gradle commands:
+
+1. **Build the application**:
+
+   ```sh
+   gradle build
+   ```
+
+2. **Run the application**:
+
+   ```sh
+   gradle run --args="https://www.graalvm.org/"
+   ```
+
+   After running the application, you should see the generated QR code printed as output.
+
+## Java Code Overview
+
+### `App.java`
+This is the main class that initializes the GraalVM context and executes the JavaScript QR code generation. The code integrates the `qrcode` NPM package by loading the JavaScript bundle (`bundle.mjs`) and calling the QR code generation function.
+
+```java
+try (Context context = Context.newBuilder("js")
```

Comment:
This is the same code from the Maven guide, but the guide itself diverges quite a bit.

It's absolutely fine if the two guides are almost identical apart from text specific to Maven, which needs to be replaced with Gradle. Could you please do that?